content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_dropship.rst

```diff
@@ -0,0 +1,147 @@
+:menuselection:`Manufacturing app --> Products --> Bills of Materials`, and select the |BoM| for the
+subcontracted product, or create a new |BoM| by clicking :guilabel:`New`.
+
+.. seealso::
+
+   For a full overview of |BoM| configuration, see the :doc:`Bill of materials
+   <../basic_setup/bill_configuration>` documentation.
+
+In the :guilabel:`BoM Type` field, select the :guilabel:`Subcontracting` option. Then, add one or
+more subcontractors in the :guilabel:`Subcontractors` field that appears below.
+
+.. image:: subcontracting_dropship/bom-type.png
+   :align: center
+   :alt: The "BoM Type" field on a BoM, configured to manufacture the product using subcontracting.
+
+Finally, make sure that all necessary components are specified on the :guilabel:`Components` tab. To
+add a new component, click :guilabel:`Add a line`, select the component in the :guilabel:`Component`
+drop-down menu, and specify the required quantity in the :guilabel:`Quantity` field.
+
+Configure Components
+--------------------
+
+To configure components for the *Dropship Subcontractor on Order* route, navigate to each component
+from the |BoM| by selecting the component's name in the :guilabel:`Components` tab, and clicking the
+:guilabel:`➡️ (right arrow)` button to the left of the name.
+
+Alternatively, navigate to each component by going to :menuselection:`Inventory app --> Products -->
+Products`, and selecting the component.
+
+On the component product form, select the :guilabel:`Purchase` tab, and add a vendor by clicking
+:guilabel:`Add a line`, selecting the vendor in the :guilabel:`Vendor` field, and adding the price
+they sell the product for in the :guilabel:`Price` field. This is the vendor that sends components
+to the subcontractor, once they are purchased.
+
+Then, click on the :guilabel:`Inventory` tab and select the :guilabel:`Dropship Subcontractor on
+Order` route in the :guilabel:`Routes` section.
+
+Repeat the process for every component that must be dropshipped to the subcontractor.
+
+Dropship subcontractor on order workflow
+========================================
+
+The dropship subcontractor on order workflow consists of three steps:
+
+#. Create a |PO| to purchase the finished product from the subcontractor (*subcontractor* |PO|);
+   doing so creates a request for quotation (RfQ) to purchase the components from the vendor, as
+   well as a receipt order.
+#. Confirm the |RfQ| to turn it into a second |PO| (*vendor* |PO|); doing so creates a *Dropship
+   Subcontractor* order.
+#. Process the *Dropship Subcontractor* order once the vendor has sent the components to the
+   subcontractor.
+#. Process the receipt once the subcontractor has finished manufacturing the subcontracted product,
+   and shipped it back to the contracting company.
+
+Create purchase order
+---------------------
+
+To create a *subcontractor* |PO|, navigate to :menuselection:`Purchase app --> Orders --> Purchase
```

Comment:
This doc is specific to workflows where a company wants to replenish their own stock of a subcontracted product, rather than have something subcontracted to meet the needs of an SO. That will be covered in the advanced workflows doc.